tests/unit/test.vcxproj

```diff
@@ -83,9 +83,11 @@
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">
     <LinkIncremental>false</LinkIncremental>
+    <LibraryPath>$(VC_LibraryPath_x64);$(WindowsSDK_LibraryPath_x64);$(VC_LibraryPath_VC_x64_Desktop)</LibraryPath>
```

Comment:
Non-blocking question: is this line one that could be put into the Directory.Build.props file in the main ebpf-for-windows directory?

Reply:
Perhaps not in this specific VS2019-related update, but definitively can go for the VS 2022 port.